spec/spec_helper.rb

```diff
@@ -56,8 +56,8 @@
   #   # is tagged with `:focus`, all examples get run. RSpec also provides
   #   # aliases for `it`, `describe`, and `context` that include `:focus`
   #   # metadata: `fit`, `fdescribe` and `fcontext`, respectively.
-  #   config.filter_run_when_matching :focus
-  #
+  config.filter_run_when_matching :focus
```

Comment:
Curious why this change?

Reply:
So I can `, :focus` tests when developing; it's a common setting

Reply:
Huh, no didn't know about that.  Looks handy